src/db.ts

```diff
@@ -230,6 +230,13 @@ export class DB {
     this._open = false;
   }
 
+  /**
+   * Get last inserted row id
+   */
+  get lastInsertRowId(): number {
+    return this._wasm.last_insert_rowid();
```

Comment:
I'm not sure on this, but do you think we should throw an error if a user attempts to obtain the last insert row id from a closed DB?

Reply:
I don't think we have to do that. If the user tries to access it when the database is closed, the value will be reset to 0 again.